Makefile

```diff
@@ -1,19 +1,210 @@
-default: ollama
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Do we still use this to build the Go project or is it only for the c++ libs now? How do I update this (will I need to?)?

Reply:
make will build ollama and any dependencies. for ollama, it'll use `go build` while dependencies will use their defined tool chains. You can update it with `cmake .`